Sourcery.xcodeproj/project.pbxproj

```diff
@@ -706,7 +706,10 @@
 				003958827841F12BBF99EB67 /* Frameworks */,
 				BF6C9BBD135B544916C5F0CD /* Pods */,
 			);
+			indentWidth = 4;
```

Comment:
is this needed?

Reply:
@ilyapuchka I think so, my personal settings are different from Sourcery's (i.e. 2 indent width; tabs, but let's not discuss that 😛), and I think contributors shouldn't have to update their preferences just to contribute - it should be part of the project itself. What do you think?
cc @krzysztofzablocki

Reply:
that's fair, anything that makes it easier to contribute